### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by 0xZ0uk** [medusajs/medusa#3259] [Feature Requests]
Been using Medusa for a week now, mostly trying it out. Really enjoy it so far, but has someone who's working his side projects late at night, I could really use a dark theme. That's why I'm contributing it!

Not going to lie, I forgot to open the feature request sooner, and I'm mostly done already... Took the liberty of reaching out to your designer Ludvig Rask, on Twitter, seeing he had some work in progress towards the a dark theme. I've worked with what he had to create a better theme.

Here's a little sneak peak for anyone interested:
![image](https://user-images.githubusercontent.com/81437712/218767764-c6a4ed77-8c55-40d9-a299-2d34f3c7c3f5.png)

**Comment by Jaaytje** [comment]
Looks good
